src/main/g8/build.sbt

```diff
@@ -1,7 +1,7 @@
 val Http4sVersion = "0.23.23"
 val CirceVersion = "0.14.5"
 val MunitVersion = "0.7.29"
-val LogbackVersion = "1.4.11"
+val LogbackVersion = "1.3.11"
```

Comment:
You didn't change it here :)

Reply:
You are right. Sorry, updated it. Pipelines should run now. Let's see :)